sql/000_base_structure.sql

```diff
@@ -248,6 +248,12 @@ CREATE TABLE `statistics_users` (
   KEY `account_id_timestamp` (`account_id`,`timestamp`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE IF NOT EXISTS `push_notification_settings` (
```

Comment:
Follow up on this, it's entirely possible to use a more generic user_settings table that include account_id, key, value as the fields. Your key could be push_notification_setting, value whatever you need to store.

This will change how we access a user account (JOIN the tables and add it to user information?) but may be better?